New changes [2017.08][2]

Realex-fire opened the pull request:
**Changes:**
1. Using TCP Fast Open for client by default.
2. Add TCP Fast Open controls for STweaks.
3. Using HYPER for sleep governor by default. (In my opinion it is more stable. For battary profile I  #used ondemand by default)
4. Removed forced removal "-siyah" prefix from kernel name.
5. Force profile reset.

**Part 1:** https://github.com/gsstudios/Dorimanx-SG2-I9100-Kernel/pull/99

Realex-fire committed:
Add TCP Fast Open controls to stweaks

Realex-fire committed:
profiles: change default sleep governor to HYPER

Realex-fire committed:
post-init: reset profiles

Realex-fire committed:
post-init: remove unnecessary code

gsstudios commented:
No objections here as well. Will wait out until the all clear as with the first PR.

Realex-fire committed:
Do not create PlayReady and DivX DRM directories.
Original commit: https://github.com/LineageOS/android_device_samsung_galaxys2-common/commit/6b314cd8d281d866adda52598c602d6871f3dc93
